test/test_api/test_environment.py

```diff
@@ -108,6 +108,13 @@ def test_scanning_venvs(venv_path):
                for venv in find_virtualenvs([parent_dir]))
 
 
+def test_find_virtualenvs_finds_via_VIRTUAL_ENV_root(venv_path):
+    """Test that is checks the directory itself, not just childs."""
+    assert "VIRTUAL_ENV" not in os.environ
+    found_envs = find_virtualenvs([venv_path])
+    assert [venv.path for venv in found_envs] == [venv_path]
```

Comment:
Note that this is very similar to the test above, which uses `dirname()`.

Good to keep both of them probably for now.